chaco/colormapped_scatterplot.py

```diff
@@ -394,10 +394,10 @@ def draw_func(x, y, size):
 
     def _color_data_changed(self, old, new):
         if old is not None:
-            old.on_trait_change(self._either_data_changed, "data_changed", remove=True)
+            old.observe(self._either_data_updated, "data_changed", remove=True)
         if new is not None:
-            new.on_trait_change(self._either_data_changed, "data_changed")
-        self._either_data_changed()
+            new.observe(self._either_data_updated, "data_changed")
+        self._either_data_updated()
```

Comment:
Hmm. I guess this method was inherited by this class? Renaming a method which is used by inherited classes can cause breakage. No changes necessary - it would be useful if we could search for users of the old method though.

Reply:
I did a search on github and there are some users. In many cases though it appears as though code was copy pasted along with the original method so this name change would not affect those users.  However, that is not always the case and some users who subclass like BaseXYPlot or ScatterPlot may use the inherited method via the old name

Reply:
@rahulporuri should we undo the name change here to be safe? Or do you think it is okay to go ahead with the change

Reply:
no i think it's fine, let's go ahead.